pom.xml

```diff
@@ -76,140 +76,13 @@
-            <groupId>com.refactoring.analysis</groupId>
-            <artifactId>org.eclipse.jdt.core_3.12.1.v20160829-0950</artifactId>
-            <version>1.0</version>
-        </dependency>
-        <dependency>
-            <groupId>com.refactoring.analysis</groupId>
-            <artifactId>org.eclipse.text_3.5.400.v20150505-1044</artifactId>
-            <version>1.0</version>
-        </dependency>
-        <dependency>
-            <groupId>com.refactoring.analysis</groupId>
-            <artifactId>org.eclipse.osgi_3.10.101.v20150820-1432</artifactId>
-            <version>1.0</version>
-        </dependency>
-        <dependency>
-            <groupId>com.refactoring.analysis</groupId>
-            <artifactId>org.eclipse.core.runtime_3.11.1.v20150903-1804</artifactId>
-            <version>1.0</version>
-        </dependency>
-        <dependency>
-            <groupId>com.refactoring.analysis</groupId>
-            <artifactId>org.eclipse.equinox.registry_3.6.0.v20150318-1503</artifactId>
-            <version>1.0</version>
-        </dependency>
-        <dependency>
-            <groupId>com.refactoring.analysis</groupId>
-            <artifactId>RefactoringMiner</artifactId>
-            <version>1.0</version>
-        </dependency>
-        <dependency>
-            <groupId>com.refactoring.analysis</groupId>
-            <artifactId>org.eclipse.equinox.app_1.3.300.v20150423-1356</artifactId>
-            <version>1.0</version>
-        </dependency>
-        <dependency>
-            <groupId>com.refactoring.analysis</groupId>
-            <artifactId>org.eclipse.equinox.common_3.7.0.v20150402-1709</artifactId>
-            <version>1.0</version>
-        </dependency>
-
-        <!-- Remote Repositories -->
-
-        <dependency>
-            <groupId>org.eclipse.jgit</groupId>
-            <artifactId>org.eclipse.jgit</artifactId>
-            <version>5.0.2.201807311906-r</version>
-        </dependency>
-
-
-        <dependency>
-            <groupId>com.jcabi</groupId>
-            <artifactId>jcabi-github</artifactId>
-            <version>0.17</version>
-        </dependency>
-
-        <dependency>
-            <groupId>com.fasterxml.jackson.core</groupId>
-            <artifactId>jackson-core</artifactId>
-            <version>2.8.4</version>
-        </dependency>
-
-        <dependency>
-            <groupId>com.fasterxml.jackson.core</groupId>
-            <artifactId>jackson-databind</artifactId>
-            <version>2.9.10.7</version>
-        </dependency>
-
-        <dependency>
-            <groupId>com.fasterxml.jackson.core</groupId>
-            <artifactId>jackson-annotations</artifactId>
-            <version>2.8.4</version>
-        </dependency>
-
-        <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-log4j12</artifactId>
-            <version>1.7.7</version>
-        </dependency>
-
-        <!-- End of RefactoringMiner and its dependencies -->
 
```

Comment:
So does the new dependency bring in all of Refactoringminer now? I think Mehran previously had a problem so that's why he had to do it this way